liquibase-core/src/main/java/liquibase/diff/compare/DatabaseObjectComparator.java

```diff
@@ -8,10 +8,10 @@
 
 public interface DatabaseObjectComparator {
 
-    final int PRIORITY_NONE = -1;
-    final int PRIORITY_DEFAULT = 1;
-    final int PRIORITY_TYPE = 5;
-    final int PRIORITY_DATABASE = 10;
+    int PRIORITY_NONE = -1;
```

Comment:
Is there any functional reason to allow callers to modify these values at runtime?

Reply:
idem above